ci/docker/install/ubuntu_onnx.sh

```diff
@@ -27,8 +27,17 @@ set -e
 set -x
 
 echo "Installing libprotobuf-dev and protobuf-compiler ..."
-apt-get install -y libprotobuf-dev protobuf-compiler
+apt-get install -y autoconf automake libtool curl make g++ unzip
+git clone --recursive https://github.com/google/protobuf.git
+cd protobuf
+git checkout 80a37e0782d2d702d52234b62dd4b9ec74fd2c95
```

Comment:
Could you elaborate on that hash? Is it stable?

Reply:
This is the hash google/protobuf used for 3.4 release - https://github.com/google/protobuf/releases/tag/v3.4.0

Reply:
Why not just do `wget https://github.com/google/protobuf/archive/v3.4.0.zip` and uzip it?

Reply:
Ah, thanks. Could you please add a comment inside the code?

@vdantu Good idea, it doesn't really make a difference, up to you.